### DISCUSSION TITLE
unexpected rows shown on Chrome console

### DISCUSSION THREAD

**Discussion by samg666** [Budibase/budibase#3198] [Help]
Hello,

When I am trying to delete a row and got errors in some reason, then I try to find out some error information for debug, and I open Chrome console and click the error links, then I saw all my rows unexpectedly. This is abnormal, I think no matter in any case, Budibase shouldn't show those rows in that link, this may leak my sensitive data. 

<img width="2589" alt="iShot2021-10-30 15 21 18" src="https://user-images.githubusercontent.com/29649651/139524287-df323b0d-74c3-43e3-a1ce-21b27183c866.png">

**Comment by mjashanks** [comment]
Hi Sam.

As far as I can tell, the delete command does not return all rows. If you can give some more steps to help investigate otherwise, than would be helpful.

However, in your screenshot, you click on the URL `/api/<tableid>/rows`. Anytime that you load a URL directly from your browser URL bar, it will do a `GET`. Therefore, your screenshot is showing the request `GET /api/<tableid>/rows`. So, I'd fully expect this to return your rows.

How you should debug this:
1. Open dev tools
2. Click on your network tab
3. Clear all previous requests (see screenshot below)
4. Click "Preserve Log", so that the requests are not cleared between page refreshes

"Clear" and "Preserve log" look like this in Chrome:
![image](https://user-images.githubusercontent.com/3524181/139653789-2972ae8b-7e91-4f8f-9b7d-363ae7824386.png)

5. Recreate your issue.
6. Now, you will likely see a few requests happen. One of them should be `DELETE /api/<tableID>/rows/<rowId>` - you can see the response of this to verify that its not returning all records.

I suspect that your action is then doing a `Navigate To`, back to your List View - which is what returns the records.

Happy to help debug further, if you'd like to provide some extra information.
